docs/usage.rst

```diff
@@ -283,8 +283,68 @@ The output of the quantized model differs slightly.
     onnx_model_4(signal, sampling_rate)
 
 
-Model with multiple nodes
--------------------------
+Custom function
+---------------
+
+So far,
+we have used
+:class:`opensmile.Smile`
+as feature extractor.
+It derives from
+:class:`audobject.Object`
+and is therefore serializable by default.
+However,
+using
+:class:`audonnx.Function`
+we can turn any function
+into a serializable object.
+For instance,
+we can define a function that extracts
+Mel-frequency cepstral coefficients (MFCCs)
+with librosa_.
+
+.. jupyter-execute::
+
+    def func(x, sr):
```

Comment:
Having https://github.com/audeering/audonnx/issues/28 in mind, I would rename here to

```suggestion
    def mfcc(x, sr):
```